app/v2/controllers/ErrorTranslator.scala

```diff
@@ -87,7 +87,7 @@ trait ErrorTranslator {
   implicit val extractionError = new Converter[ExtractionError] {
```

Comment:
We don't, as we don't actually use the result in returning the status to the use (we fire and forget).

Good question though -- should we be failing the call if an association fails? I suppose it can fail for not having a box at all (rare, but possible, I guess), or just an overall failure.